### ISSUE TITLE
Logging?

### ISSUE THREAD

**Issue by jmbell1989** [aler9/rtsp-simple-server#37]
Hey, really like the work you've put it. Super easy and straightforward. I have a question though, is there a way to log the data that outputted to a file or something. I'd like to monitor these interactions. I saw there is a prof setting to monitor performance. Not sure if that's what I'm looking for or not but just dumping the output to a file would suffice.

Thanks for the help!

**Comment by benbullnz** [comment]
Hey, I'm definitely no expert but I made this work by piping the console output to a log file. I know there is higher level logging available, someone else can talk to that. 

In the meantime, kill your server and then restart rtsp-simple-server by running the following command:
./rtsp-simple-server |& tee server.log

This will create a server.log file in the same directory as rtsp-simple-server.

You can view the log file by running:
cat server.log
or//
tail -f server.log (this will continually update)

PS, i'm no expert so there might be a better way - but this worked for me.

**Comment by jmbell1989** [comment]
I am unfortunately on Windows.. should've mentioned that:/ Thanks for the suggestion though. That will most definitely work in Linux

**Comment by aler9** [comment]
Hello, regarding monitoring, i think that two separate new features could be implemented:
* an option to log to file instead than logging to stdout, expecially for Windows users that do not want to use console features
* a metrics exporter, compatible with Prometheus, that provides via HTTP the number of connected clients, publishers, readers

Regarding pprof, i added it to monitor RAM and CPU usage; for instance, after enabling the option, you can get CPU
```
docker run --rm -it --network=host golang:1.14-alpine3.12 go tool pprof -text http://localhost:9999/debug/pprof/profile?seconds=30
```
or RAM
```
docker run --rm -it --network=host golang:1.14-alpine3.12 go tool pprof -text http://localhost:9999/debug/pprof/heap
```

**Comment by aler9** [comment]
both features are available in v0.9.6.
Please look at the configuration file and the README for instructions on how to use both.
